mass/adapters/outbound/dao.py

```diff
@@ -64,3 +67,27 @@ def get_dao(self, *, class_name: str) -> ResourceDao:
             return self._resource_daos[class_name]
         except KeyError as err:
             raise DaoNotFoundError(class_name=class_name) from err
+
+    def collection_init_and_index_creation(self) -> None:
```

Comment:
Maybe call it `create_collections_and_indexes_if_needed` to make the purpose clearer.

Reply:
Changed in [e8b9e8d](https://github.com/ghga-de/mass/pull/13/commits/e8b9e8df779ccd22b734db53f296f943bec92bf8)